<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DataGen</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="ExcelTableView" name="m_listCtrl">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>10</y>
     <width>406</width>
     <height>401</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>430</y>
     <width>401</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>401</width>
     <height>20</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QCheckBox" name="ck_lua">
      <property name="text">
       <string>Lua</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="ck_json">
      <property name="text">
       <string>Json</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="ck_php">
      <property name="text">
       <string>Php</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>500</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QPushButton" name="btnStart">
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnStopCurrentTask">
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_config">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Config</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnAboutVersion">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>About</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnCloseDlg">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>446</y>
     <width>401</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLineEdit" name="m_outPath">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnOutPath">
      <property name="text">
       <string>Out Dir</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>410</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="btnAllSelect">
      <property name="text">
       <string>Select All</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnAllUnSelect">
      <property name="text">
       <string>Un Select All</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnClearAll">
      <property name="text">
       <string>Clear All</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnOpenFile">
      <property name="text">
       <string>Add File</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>m_listCtrl</zorder>
  <zorder>line</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>ck_php</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ExcelTableView</class>
   <extends>QTableView</extends>
   <header location="global">src/exceltableview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
